tests/SnsServiceProviderTest.php

```diff
@@ -0,0 +1,82 @@
+<?php
+
+namespace NotificationChannels\AwsSns\Test;
+
+use Aws\Sns\SnsClient as SnsService;
+use Illuminate\Contracts\Foundation\Application;
+use Mockery;
+use Mockery\Adapter\Phpunit\MockeryTestCase;
+use NotificationChannels\AwsSns\Sns;
+use NotificationChannels\AwsSns\SnsChannel;
+use NotificationChannels\AwsSns\SnsServiceProvider;
+
+class SnsServiceProviderTest extends MockeryTestCase
```

Comment:
Try extending the default `TestCase` and using the mockery trait `use MockeryPHPUnitIntegration;`